util/cache.cc

```diff
@@ -390,8 +390,8 @@ Cache::Handle* LRUCache::Insert(
   // Allocate the memory here outside of the mutex
   // If the cache is full, we'll have to release it
   // It shouldn't happen very often though.
-  LRUHandle* e =
-      reinterpret_cast<LRUHandle*>(malloc(sizeof(LRUHandle) - 1 + key.size()));
+  LRUHandle* e = reinterpret_cast<LRUHandle*>(
+                    new char[sizeof(LRUHandle) - 1 + key.size()]);
```

Comment:
I'm not expert on this but does it change any alignment?

Hope about build a rocksdb_allocate() and rocksdb_free() in port and we implement differently in Windows and Linux?